From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Stephen A Pohl <spohl.mozilla.bugs@gmail.com>
Date: Wed, 26 Feb 2020 20:32:13 +0000
Subject: [PATCH 2/2] Bug 1583854: Allow builds with the 10.15 macOS SDK.
 r=mshal

Depends on D64447

Differential Revision: https://phabricator.services.mozilla.com/D64448

--HG--
extra : moz-landing-system : lando

diff --git a/build/moz.configure/toolchain.configure b/build/moz.configure/toolchain.configure
index 4957278cc90d..fc5d2bdf8342 100755
--- a/build/moz.configure/toolchain.configure
+++ b/build/moz.configure/toolchain.configure
@@ -141,7 +141,7 @@ with only_when(host_is_osx | target_is_osx):
     @imports(_from='biplist', _import='readPlist')
     def macos_sdk(sdk, host):
         sdk_min_version = Version('10.11')
-        sdk_max_version = Version('10.14')
+        sdk_max_version = Version('10.15')
 
         if sdk:
             sdk = sdk[0]
